src/list-items/list-item-label.html

```diff
@@ -60,10 +60,11 @@
         display: inline-block;
         padding: 3px 6px;
         border-radius: 6px;
-        color: var(--primary-background-color);
-        color: var(--tag-text-color);
-        background-color: var(--pr-status-color);
-        background-color: var(--tag-color);
+        color: var(--primary-background-color, --tag-text-color);
+        background-color: var(--pr-status-color, --tag-color);
+        overflow: hidden;
+        text-overflow: ellipsis;
+        max-width: 120%;
```

Comment:
I'm afraid this is not a very good fix. The `list-item-label` is used in many other places and this change will restrict the size of the labels everywhere. Furthermore, 120% is an arbitrary number that could still not work on other screens than the ones we test. I'd think you can fix this in the place where `list-item-label` is used.